.github/workflows/block-fixup-commits.yaml

```diff
@@ -0,0 +1,30 @@
+name: Block "fixup" commit merges
+
+on:
+  workflow_call:
+
+jobs:
+  check-commits:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        fetch-depth: ${{ github.event.pull_request.commits }} + 1
+
+    - name: Fetch the base branch
+      run:  git fetch origin ${{ github.event.pull_request.base.ref }}
+
+    - name: Look for the fixup prefix
+      id: search-for-fixup-prefix
+      run: |
+        commit_messages=$(git log origin/${{ github.event.pull_request.base.ref }}..$GITHUB_HEAD_REF --format=%B)
+        fixup_commits=$(echo "$commit_messages" | grep "^fixup")
+        if [ -n "$fixup_commits" ]; then
+            echo "Please make sure that all commits that have the 'fixup' prefix need to be squashed before merging."
+            echo "Commits with 'fixup' prefix:"
+            echo "$fixup_commits"
+            exit 1
+        fi
```

Comment:
```suggestion
        fi

```
Newline :)

Reply:
Fixed in the [forcepush](https://github.com/python-discord/.github/commit/f2f1f2db0b1dcb2f44f0a619d384c9118e5be983)